src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -494,6 +549,8 @@ function loseFocus() {
     ctaAreaElementFocusable.blur();
   }
 }
+
+const hasLinksInItems = computed(() => props.items.some(item => !!item.to || !!item.href));
```

Comment:
```suggestion
const hasLinksInItems = computed(() => props.items.some(item => item.to || item.href));
```
Casting to boolean will be proceeding for us by js as these values are used in logical operation ||.

Reply:
corrected, thanks